drools-ansible-rulebook-integration-main/src/main/java/org/drools/ansible/rulebook/integration/main/Payload.java

```diff
@@ -22,6 +20,9 @@ public class Payload {
 
     private int shutdown = 0;
 
+    // set true when matchedEvents occupies too much memory
+    private boolean discardMatchedEvents = false;
```

Comment:
I need to add this to avoid memory retention on `Payload` side in the test cases.